pandasql/core.py

```diff
@@ -111,6 +112,13 @@ def _compute_sqlite(self):
 
         return self.result
 
+    def memory_usage(self):
+        if isinstance(self._cached_result, pd.DataFrame):
+            return self._cached_result.memory_usage(deep=True, index=True).sum()
+        elif isinstance(self._cached_result, pd.Series):
+            return self._cached_result.memory_usage(deep=True, index=True)
+        # TODO: handle Aggregator, GroupByDataFrame, and GroupByProjection
```

Comment:
Should this be returning something in the default case?

Reply:
I think None should be easiest, as it indicates we don't know, as opposed to 0 or something else.

Reply:
Would a `NotImplemented` be useful?